packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+import 'package:flutter/foundation.dart';
```

Comment:
blank line between dart: imports and package: imports
dart:ui should always be imported `as ui` or, for the most common classes, should explicitly say `show Color` et al.